### ISSUE TITLE
Use ensemble to start Python Backend and PyTorch Backend, prompting not supported for Pytorch Backend.

### ISSUE THREAD

**Issue by 756890911** [triton-inference-server/server#3915]
I use ensemble to start Python Backend and PyTorch Backend and say not supported for Pytorch Backend.

Python Backend does preprocessing and PyTorch Backend does model prediction. Skip Model configuration auto-complete for 'AIp-infer -coco-mask': not supported for pytorch backend

使用
/opt/tritonserver/bin/tritonserver \
--model-repository=/ningzhou.li/ensembled_image_new \
--strict-model-config=false \
--exit-timeout-secs=20 \
--cuda-memory-pool-byte-size=0:2147483648
启动

**Comment by tanmayv25** [comment]
The autocomplete feature is not supported for pytorch models and user must provide complete config.pbtxt for the model. Because pytorch model do not carry sufficient metadata we can not populate the model configuration for pytorch models in our pytorch backend.

For more details about `--strict-model-config=false` option, see here: https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration

> Specifically, TensorRT, TensorFlow saved-model, and ONNX models do not require a model configuration file because Triton can derive all the required settings automatically. All other model types must provide a model configuration file.

**Comment by 756890911** [comment]
I set the { key: "FORCE_CPU_ONLY_INPUT_TENSORS" value: {string_value:"no"}} in CONFIG.PBTXT  Solved the problem.

> By default, The Python backend moves all input tensors to CPU before providing them to the Python model. Starting from 21.09, you can change this default behavior. By setting FORCE_CPU_ONLY_INPUT_TENSORS to "no", Triton will not move input tensors to CPU for the Python model. Instead, Triton will provide the input tensors to the Python model in either CPU or GPU memory, depending on how those tensors were last used. You cannot predict which memory will be used for each input tensor so your Python model must be able to handle tensors in both CPU and GPU memory. To enable this setting, you need to add this setting to the parameters section of model configuration:
